ec3730/Views/Host/HostViewSectionFocusView.swift

```diff
@@ -10,7 +10,7 @@ struct HostViewSectionFocusView: View {
                 ScrollView {
                     HostViewSectionContent(sectionModel: model, canQuery: true)
                 }
-                HostBarView(url: url, date: date)
+                HostBarView(url: url, date: date).background(Color(UIColor.systemGroupedBackground))
```

Comment:
Why is this change necessary?

Reply:
HostBarView(url: url, date: date).background(Color(UIColor.systemGroupedBackground))

[https://we.tl/t-FPltFCM832](url) Without background color , it will show like the given screenshot, there will be no background color.

Reply:
Shouldn't this just ignore the safe area?

Reply:
[https://we.tl/t-2KPswENwxO](url) attached screenshots , in case we use only the ignore safe area , or even without that there is no background color, but if we want background color , then we need to give background color